app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,7 @@
+class FoodsController < ApplicationController
+  def show
+    @foods = FoodsFacade.foods(params[:q])
+    @results = FoodsFacade.results_number(params[:q])
```

Comment:
Nice job breaking these two requirements into separate methods at the facade level.